src/test/app/claims/claimModelConverter.ts

```diff
@@ -65,4 +72,13 @@ describe('ClaimModelConverter', () => {
     const converted: ClaimData = ClaimModelConverter.convert(claimDraft)
     expect(converted.interest.interestDate).to.be.undefined
   })
+
+  it('should not contain title if blank', () => {
+    const claimDraft = prepareClaimDraft(theirIndividualDetails, individualDefendant)
+    // @ts-ignore
```

Comment:
why the ts-ignores? can we not write it in a way to pass the linter?

Reply:
Because being it a test my laziness took over and I let intellij put the ignores, rather than spending literally 3 seconds trying to appease the linter properly. Sorry :D